examples/ethereum-tracker/Cargo.toml

```diff
@@ -5,18 +5,18 @@ authors = ["Linera <contact@linera.io>"]
 edition = "2021"
 
 [dependencies]
-alloy = { workspace = true, default-features = false, features = [ "rpc-types-eth" ] }
-async-graphql.workspace = true
-linera-ethereum.workspace = true
-linera-sdk = { workspace = true, features = [ "ethereum" ]}
-serde.workspace = true
+alloy = { workspace = true, default-features = false, features = ["rpc-types-eth"] }
+async-graphql = { workspace = true }
+linera-ethereum = { workspace = true }
+linera-sdk = { workspace = true, features = [ "ethereum" ] }
+serde = { workspace = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
 linera-sdk = { workspace = true, features = ["test", "wasmer", "ethereum"] }
 tokio = { workspace = true, features = ["rt", "sync"] }
 
 [dev-dependencies]
-assert_matches.workspace = true
+assert_matches = { workspace = true }
```

Comment:
We're using the other syntax (`assert_matches.workspace = true`) everywhere else.

Reply:
Yes, it was `cargo sort` problems.